test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -384,6 +385,11 @@ bool IsCompressionTest()
             }
         }
 
+        protected virtual void OnCreatingNewDatabase(DatabaseRecord databaseRecord)
```

Comment:
isn't that the same as options.ModifyDatabaseRecord?

Reply:
Yes, but that is `ModifyDatabaseRecord` needs to be set on each options call, where as this allows us to set it up at the class level.